README.md

```diff
@@ -54,6 +54,7 @@ A curated list of all things anime - streaming/downloading/reading/tracking/foru
 * [anime-loads.org](https://www.anime-loads.org/) - Contains German Dubs/Subs
 * [animepahe.com](https://animepahe.com/)
 * [anime-planet.com](https://www.anime-planet.com/)
+* [animixplay.to](https://animixplay.to/)
```

Comment:
this doesn't work

Reply:
wdym it doesn't work hmm

Reply:
this site works for me in India tho

Reply:
then I think just blocked in Japan, @toprak can you check?

Reply:
then I think just blocked in Japan, @toprak can you check?

Reply:
@anshumanv sorry for very late reply, i have been busy with school thing anyways it works for me

Reply:
That was quite a time taking thing 🤧

Reply:
@LiReNa00 yes sorry about that i got many problems also too busy with school i dont have life anymore :D

Reply:
@toprak that is very relatable 🤧